### ISSUE TITLE
ClusterServiceVersion Status: Failed

### ISSUE THREAD

**Issue by leszko** [operator-framework/operator-lifecycle-manager#769]
In the `latest` version of OLM, when any ClusterServiceVersion is uploaded, the status is always failed. The [latest released version](https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v3.11.0) (3.11.0) works fine.

![image](https://user-images.githubusercontent.com/2834997/54620218-1e7ae180-4a66-11e9-843a-0825112acb49.png)

**Comment by njhale** [comment]
@leszko Thank you for submitting this issue. Could you collect the CSV's status resource and post it here as well?

**Comment by leszko** [comment]
Ok, I probably see the potential issue:
```
$ kubectl get pods -n olm
NAME                                READY   STATUS             RESTARTS   AGE
catalog-operator-5cb6b85798-phdf4   1/1     Running            0          10m
olm-operator-84754d9fb9-8wwb4       0/1     CrashLoopBackOff   6          10m
olm-operators-ftxbc                 1/1     Running            0          10m

$ kubectl logs olm-operator-84754d9fb9-8wwb4 -n olm
flag provided but not defined: -writeStatusName
Usage of /bin/olm:
  -alsologtostderr
        log to standard error as well as files
  -debug
        use debug log level
  -interval duration
        wake up interval (default 5m0s)
  -kubeconfig string
        absolute path to the kubeconfig file
  -log_backtrace_at value
        when logging hits line file:N, emit a stack trace
  -log_dir string
        If non-empty, write log files in this directory
  -logtostderr
        log to standard error instead of files
  -stderrthreshold value
        logs at or above this threshold go to stderr
  -v value
        log level for V logs
  -version
        displays olm version
  -vmodule value
        comma-separated list of pattern=N settings for file-filtered logging
  -watchedNamespaces -watchedNamespaces=""
        comma separated list of namespaces for olm operator to watch. If not set, or set to the empty string (e.g. -watchedNamespaces=""), olm operator will watch all namespaces in the cluster.
```

Probably something is wrong with your OLM Operator Image: `quay.io/coreos/olm@sha256:995a181839f301585a0e115c083619b6d73812c58a8444d7b13b8e407010325f`. Could you please check?

**Comment by njhale** [comment]
How are you installing OLM? Which set of manifests are you using?

**Comment by leszko** [comment]
Yes, the issue is resolved by #771 . Thanks!
